src/nbl/ext/ImGui/ImGui.cpp

```diff
@@ -548,43 +629,66 @@ namespace nbl::ext::imgui
 		}
 	}
 
-	//-------------------------------------------------------------------------------------------------
-	// TODO: Handle mouse cursor for nabla
-	//static void UpdateMouseCursor()
-	//{
-		//auto & io = ImGui::GetIO();
-
-		//if (io.ConfigFlags & ImGuiConfigFlags_NoMouseCursorChange)
-		//{
-		//    return;
-		//}
-		//ImGuiMouseCursor imgui_cursor = ImGui::GetMouseCursor();
-		//if (io.MouseDrawCursor || imgui_cursor == ImGuiMouseCursor_None)
-		//{
-		//    // Hide OS mouse cursor if imgui is drawing it or if it wants no cursor
-		//    MSDL::SDL_ShowCursor(MSDL::SDL_FALSE);
-		//}
-		//else
-		//{
-		//    // Show OS mouse cursor
-		//    MSDL::SDL_SetCursor(mouseCursors[imgui_cursor] ? mouseCursors[imgui_cursor] : mouseCursors[ImGuiMouseCursor_Arrow]);
-		//    MSDL::SDL_ShowCursor(MSDL::SDL_TRUE);
-		//}
-	//}
-
-	bool UI::Render(IGPUCommandBuffer* commandBuffer, int const frameIndex)
+	void UI::createMDIBuffer(const uint32_t totalByteSize)
+	{
+		constexpr static core::bitflag<IDeviceMemoryAllocation::E_MEMORY_ALLOCATE_FLAGS> allocateFlags(IDeviceMemoryAllocation::EMAF_DEVICE_ADDRESS_BIT);
+		constexpr static uint32_t minStreamingBufferAllocationSize = 4u,
+			maxStreamingBufferAllocationAlignment = 1024u * 64u;
```

Comment:
keep `minStreamingBufferAllocationSize` above 32 otherwise you end up using more reserved memory to keep allocator state than the allocator arena size

Reply:
resolved